plugins/scaffolder-actions/scaffolder-backend-module-utils/src/actions/sleep.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createTemplateAction } from '@backstage/plugin-scaffolder-backend';
+import { InputError } from '@backstage/errors';
+
+export function createSleepAction() {
```

Comment:
As a compromise, can we pass in a max sleep here and have it default to undefined?

createSleepAction(options: { maxSleep: number })

Reply:
Done